typings/index.d.ts

```diff
@@ -30,8 +30,8 @@ declare module 'discord-player' {
 
         public static get AudioFilters(): PlayerFilters;
         public isPlaying(message: Message): boolean;
-        public setFilters(message: Message, newFilters: Partial<Filters>): Promise<void>;
-        public play(message: Message, query: string | Track, firstResult?: boolean): Promise<void>;
+        public setFilters(message: Message, newFilters: Partial<FiltersOption>): Promise<void>;
```

Comment:
Hmm no it is still a Partial<Filters> right? Users can use:
```js
setFilters(messave, {
	gate: 'agate'
});
```

Reply:
I don't know what you're trying to say to me.